### PR TITLE
[Bug] Fix  TimeStamp convert bug in CarbonExample

### PR DESCRIPTION
**Pull request by Jay357089**
issue #697

### CODE REVIEW

File: examples/src/main/scala/org/carbondata/examples/util/InitForExamples.scala [hunk 1]
```diff
@@ -48,6 +49,8 @@ object InitForExamples {
     // true -> use table split partition, support multiple partition loading
     // false -> use node split partition, support data load by host partition
     CarbonProperties.getInstance().addProperty("carbon.table.split.partition.enable", "false")
+    CarbonProperties.getInstance().addProperty(
+      CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, "yyyy/mm/dd")
     cc
   }
```
Comment (foryou2030) [inline examples/src/main/scala/org/carbondata/examples/util/InitForExamples.scala]: it maybe not proper, InitForExamples have been called by other examples

Comment (Jay357089) [inline examples/src/main/scala/org/carbondata/examples/util/InitForExamples.scala]: @foryou2030 handled.

**Comment by Jay357089** [comment]
@jackylk @QiangCai pls review.

**Comment by QiangCai** [comment]
LGTM

**Pull request merged**